hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md

```diff
@@ -1069,6 +1113,14 @@ java.io.IOException: Invalid region specified "iceland-2":
 
 The region specified in `fs.s3a.s3guard.ddb.region` is invalid.
 
+### `MetadataPersistenceException`
+
+An filesystem write operation failed due to failure to persist metadata to
```

Comment:
I'd suggest a wording change here: "An filesystem write operation failed due to failure to persist metadata to S3Guard." -> "A filesystem write operation failed to persist metadata to S3Guard."

Reply:
Agreed.  Updated in https://github.com/apache/hadoop/pull/666/commits/8ff542741ab3386a9ce967fb2f36fab2265413f4